tests/unit/libbpf_test.cpp

```diff
@@ -3124,3 +3124,82 @@ TEST_CASE("bind_tail_call_max_exceed", "[libbpf]")
 
     usersim_trace_logging_set_enabled(false, 0, 0);
 }
+
+TEST_CASE("libbpf map batch", "[libbpf]")
```

Comment:
also add negative test cases, like:
* update_batch_size = 0 on input to bpf_map_update_batch
* fetched_batch_size = 0 on input to bpf_map_lookup_batch
* elem_flags contains extra bits set that have no meaning
* invalid map fd